posthog/api/search.py

```diff
@@ -55,6 +55,11 @@
         "search_fields": {"name": "A"},
         "extra_fields": ["name"],
     },
+    "survey": {
+        "klass": Survey,
+        "search_fields": {"name": "A", "description": "C"},
```

Comment:
This searches the fields name with weighting "A" and description with weighting "C", same as we do for other models.